sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/DangerousGetHandleShouldNotBeCalled.cs

```diff
@@ -29,19 +30,17 @@ namespace SonarAnalyzer.Rules.CSharp
 {
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     [Rule(DiagnosticId)]
-    public sealed class DoNotCallDangerousGetHandleMethod : DoNotCallMethodsBase
+    public sealed class DangerousGetHandleShouldNotBeCalled : DangerousGetHandleShouldNotBeCalledBase<InvocationExpressionSyntax>
```

Comment:
Is it normal that this class doesn't inherit the CSharp base class?

Reply:
`DangerousGetHandleShouldNotBeCalledBase` (which inherits `DoNotCallMethodsBase`)  is common for the CSharp and VB.net implementations.

In order to use `DoNotCallMethodsCSharpBase` and also use the common code from `DangerousGetHandleShouldNotBeCalledBase`, I would have to use an inner class inheriting `DoNotCallMethodsBase`, and making that refactoring for all the rules (that are currently implemented only in C#) does not make sense at the moment.